### PR TITLE
add_author_gh

### PR DESCRIPTION
**Pull request by maelle**
cf #71

### CODE REVIEW

**Comment by maelle** [comment]
I added (very limited) doc for the full name parsing too, but for `add_me` it could go away later if #72 is a good idea.

**Comment by maelle** [comment]
And I didn't use "via" in the names of the function/method because it was too long.

**Comment by maelle** [comment]
[This commit to a package of mine](https://github.com/ropenscilabs/roblog/commit/4243c34d72a998d050f8aece25b889ef11502e72) is the result of my running `desc::desc_add_author_gh("aedobbyn")` ✨

File: R/description.R [hunk 1]
```diff
@@ -308,13 +308,18 @@ desc <- function(cmd = NULL, file = NULL, text = NULL, package = NULL) {
 #' \preformatted{  description$add_author(given = NULL, family = NULL, email = NULL,
 #'     role = NULL, comment = NULL, orcid = NULL)
 #'   description$add_me(role = "ctb", comment = NULL, orcid = NULL)
+#'   description$add_author_gh(role = "ctb", username = "jeroen")
```
[the remark this answers is not in the document]
Comment (maelle) [reply R/description.R]: You mean it needs to become
```r
    add_author_gh = function(username, role = "ctb", comment = NULL, orcid = NULL)
      idesc_add_author_gh(self, private, role = role,
                   username = username,
                   comment = comment, orcid = orcid)
```
?

[the remark this answers is not in the document]
Comment (gaborcsardi) [reply R/description.R]: No, just
```r
description$add_author_gh(username, role = "ctb", comment = NULL, orcid = NULL)
```

[the remark this answers is not in the document]
Comment (maelle) [reply R/description.R]: Fixed!

**Comment by gaborcsardi** [comment]
Thanks!

**Comment by maelle** [comment]
🎉

**Pull request merged**